launch/test/counter.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+
+import argparse
+import signal
+import sys
+import time
+
+
+def main(argv=sys.argv[1:]):
+    parser = argparse.ArgumentParser(
+        description=(
+            'Simple program outputting a counter. '
+            'Even values go to STDERR, odd values go to STDOUT.'))
```

Comment:
And add a backslash after counter.' Multiline strings without \ tend to be prone to breakage.